lib/api/authn/force-user.js

```diff
@@ -0,0 +1,19 @@
+const debug = require('../../debug').authentication
+
+/**
+ * Enforces the `--force-user` server flag, hardcoding a webid for all requests,
+ * for testing purposes.
+ */
+function initialize (app, argv) {
+  const forceUserId = argv.forceUser
+  app.use('/', (req, res, next) => {
+    debug(`Identified user (override): ${forceUserId}`)
+    req.session.userId = forceUserId
+    res.set('User', forceUserId)
```

Comment:
It was, yeah. I think we just forgot to remove it here. 
So, yes, it's useful to fake TLS authentication, but that should still be taken care of in the webid-tls.js `handler()` function. (As in, forceUser id sets the session.userId, and the TLS handler short-circuits on there being a session.userId already).